src/libs/structs/config.rs

```diff
@@ -1,24 +1,21 @@
 use lazy_static::lazy_static;
-use serde_derive::Deserialize;
+use serde::Deserialize;
 use std::sync::Mutex;
 lazy_static! {
 	pub static ref CONFIG: Mutex<Config> = Mutex::new(Config::default());
 }
 
-#[derive(Debug, Deserialize)]
-pub struct AutostartCmd {
-	pub cmd: String,
-}
-
 #[derive(Debug, Deserialize, Default)]
 pub struct Autostart {
-	pub cmd: Vec<AutostartCmd>,
+	pub cmd: Vec<String>,
 }
 
 #[derive(Debug, Default, Deserialize)]
 pub struct General {
 	pub workspaces: u8,
+    #[serde(rename="gaps_in")]
```

Comment:
Maybe we should use the same names on both sides? I think `gaps_in` sounds better.

Same for the fields below.